deepmd/dpmodel/fitting/invar_fitting.py

```diff
@@ -139,6 +139,7 @@ def __init__(
             raise NotImplementedError("atom_ener is not implemented")
 
         self.dim_out = dim_out
+        self.atom_ener = (atom_ener,)
```

Comment:
im confused by the assignment. it gives a `Tuple[List[float]]`?

Reply:
typo, reverted.